coding-concepts.js

```diff
@@ -8,48 +8,48 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+ //console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:its gonna say how many data types there now is in the array.
+// b) Verify and explain: The output was "5" which accurately shows that there is now 5 strings in the array. 
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: the code is going to ouput the numbers of charactors and spaces inside the string. 
+// b) Verify and explain: the output was 10 which is the length if you count "learn" the space " " and "2023" which is 10 total.
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+//console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It should output an "o" which is the charactor that is in the 4th index since the count starts at zero. 
+// b) Verify and explain:the output was o which means my assumption was in line. 
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+ //console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: the index is counting out the datatypes in the array. so the output should be "ruby" since index counts always start at 0.
+// b) Verify and explain:The output was Ruby and the code performed as expexted. 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+ //console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:The code will change the values in the array into capital letters. 
+// b) Verify and explain:the code outputed a Type error. It did not run. 
 
 // --------------------5) What will this log?
 
 const dataTypes = ["number", "string", "Boolean", "undefined"]
-// console.log(typeof dataTypes.length)
+console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:The output will be a 4 since that is the number of data types in the array.
+// b) Verify and explain:The output was "number" and im honestly confused as to why that is and will ask for clarity. 
```

Comment:
If you ran the code for `dataTypes.length` you would see it return 4. So really what we are doing is logging `typeof 4` which will output the name of the data type 4 belongs to which is number.